tools/analysis/slither/log/read_storage_error.md

```diff
@@ -0,0 +1,16 @@
+```
```

Comment:
Same comment as the manticore error README

Reply:
+1